src/Symfony/Component/Console/Input/Input.php

```diff
@@ -85,6 +86,35 @@ public function getArguments(): array
         return array_merge($this->definition->getArgumentDefaults(), $this->arguments);
     }
 
+    /**
+     * Returns all the given arguments NOT merged with the default values.
```

Comment:
I do not know if the description should be kept. I don't think I can add a strict equivalent when using `@method` to `InputInterface` hence might be worth to keep it there for having the complete doc when adding the method to the interface instead